CMakeLists.txt

```diff
@@ -16,7 +16,8 @@ option(FACELIFT_ENABLE_CODEGEN "Enable code generator build and install. Should
 option(FACELIFT_ENABLE_DBUS_IPC "Force DBus IPC support (no autodetection)" OFF)
 option(FACELIFT_DISABLE_DBUS_IPC "Force disable DBus IPC support (no autodetection)" OFF)
 option(FACELIFT_BUILD_EXAMPLES "Enable build of examples" ON)
-option(FACELIFT_BUILD_TEST "Enable build of tests" OFF)
+option(FACELIFT_BUILD_TESTS "Enable build of tests" OFF)
+option(FACELIFT_BUILD_OBSOLETE_TEST "Enable build of obsolete tests" OFF)
```

Comment:
I would prefer to keep all tests under one option. We can remove any obsolete test later